File: src/com/prey/actions/observer/ActionResult.java
/*******************************************************************************
 * Created by Carlos Yaconi.
 * Copyright 2011 Fork Ltd. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.actions.observer;

import com.prey.actions.HttpDataService;

public class ActionResult {

	private String result;
	private HttpDataService dataToSend;

	public String getResult() {
		return result;
	}

	public void setResult(String result) {
		this.result = result;
	}

	public HttpDataService getDataToSend() {
		return dataToSend;
	}

	public void setDataToSend(HttpDataService dataToSend) {
		this.dataToSend = dataToSend;
	}

}

File: app/src/main/java/com/prey/actions/observer/ActionResult.kt
/*******************************************************************************
 * Created by Orlando Aliaga
 * Copyright 2025 Prey Inc. All rights reserved.
 * License: GPLv3
 * Full license at "/LICENSE"
 ******************************************************************************/
package com.prey.actions.observer

import com.prey.actions.HttpDataService

/**
 * Represents the result of an action, including the outcome and any data to be sent.
 */
class ActionResult {
    /**
     * The result of the action, represented as a string.
     */
    var result: String? = null

    /**
     * The data to be sent as a result of the action, represented as an HttpDataService object.
     */

    var dataToSend: HttpDataService? = null

}